tools/spell/find-unknown-comment-words

```diff
@@ -109,15 +109,6 @@ do
         then
             continue
         fi
-        # Disable spell check for submoduled libraries.
-        if [[ $file == *"coreMQTT"* ]]
-        then
-            continue
-        fi
-        if [[ $file == *"coreHTTP"* ]]
-        then
-            continue
-        fi
```

Comment:
Why are these removed? The reason for these to be added was to disable spell check in submoduled repos. The spell check for those will be done in those repos.

Reply:
"We updated the CI to check only the platform/ demos/ and integration-test/ directories."